features/FEATURE_LWIP/lwip-interface/ppp_lwip.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2017 ARM Limited. All rights reserved.
+ */
+
+#ifndef PPP_LWIP_H_
+#define PPP_LWIP_H_
+
+#include "FileHandle.h"
+#include "NetworkStack.h"
+
+namespace mbed {
+
+void ppp_bringup_test();
```

Comment:
no docs for these functions? 

``ppp_bringup_test`` cannot spot an implementation within this commit